easybuild/easyconfigs/c/CNVnator/CNVnator-0.3.3-foss-2016b.eb

```diff
@@ -0,0 +1,40 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   Copyright 2014-2017 adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Sepulveda <exequiel.sepulveda@gmail.com>
+# License::     CCPL
+#
+# Notes::
+##
+
+easyblock = 'MakeCp'
+
+name = 'CNVnator'
+version = '0.3.3'
+
+homepage = 'https://github.com/abyzovlab/CNVnator'
+description = """ a tool for CNV discovery and genotyping from depth-of-coverage by mapped reads
+"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = ['v%(version)s/%(name)s_v%(version)s.zip']
+source_urls = ['https://github.com/abyzovlab/CNVnator/releases/download']
+
+dependencies = [
+    ('Perl', '5.24.0'),
+]
+
+skipsteps = ['build']
```

Comment:
@boegel Hmm, there are many other files inside the .zip like src, license etc. a perl script is really what is needed here.